### DISCUSSION TITLE
Is this theme still being developed?

### DISCUSSION THREAD

**Discussion by Josh65-2201** [purogamer/Fluent-for-Steam#93] [Q&A] [answered]
There's hasn't been any activity on this as far as I can tell.

**Comment by Arrdee81** [comment]
Doesn't appear to be being developed.  Shame really, this was cool.

**Comment by BlankShade98** [comment, marked as answer]
It is, but most of the updates are being published either on the dev's Patreon or in the skin's Discord server.
The Patreon link is available here on GitHub, on the skin's homepage.
And here's the Discord invite link: https://discord.gg/gwmXtUjHGd
